clar2wasm/tests/wasm-generation/sequences.rs

```diff
@@ -120,3 +120,17 @@ fn extract_sequence(sequence: PropValue) -> SequenceData {
         _ => panic!("Should only call this function on the result of PropValue::any_sequence"),
     }
 }
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crosscheck_map_is_eq(val in (0usize..=20).prop_flat_map(PropValue::any_sequence)) {
+        let snippet = format!("(and (fold or (map not (list (is-eq {val} {val}) (is-eq {val} {val}))) true))");
```

Comment:
I'm not convinced by this test: it uses `map` but doesn't really test it. The result should be true for sure, but what provokes the actual `true` here? Are we sure that `(map not (list (is-eq {val} {val}) (is-eq {val} {val})))` is not equal to `(list true true)` or `(list true false)` or `(list false true)`?

Reply:
Add two different tests that has `map` as the test focus.